pkgs/applications/misc/gpxsee/default.nix

```diff
@@ -1,17 +1,17 @@
-{ stdenv, fetchFromGitHub, qmake, qttools, makeWrapper }:
+{ stdenv, fetchFromGitHub, qmake, qttools, wrapQtAppsHook }:
 
 stdenv.mkDerivation rec {
   pname = "gpxsee";
-  version = "7.9";
+  version = "7.11";
 
   src = fetchFromGitHub {
     owner = "tumic0";
     repo = "GPXSee";
     rev = version;
-    sha256 = "029l5dhc9nnxiw7p0s4gyfkcqw709z7lz96aq8krs75mfk4fv07k";
+    sha256 = "1b4ky7m990h3rmam9lb1w6vns1mxd8ri6is3a8qgdl8kd6xcl5d7";
   };
 
-  nativeBuildInputs = [ qmake makeWrapper ];
+  nativeBuildInputs = [ qmake wrapQtAppsHook ];
```

Comment:
This is a qt application and it should be using `mkDerivation` instead of `stdenv.mkDerivation` which will automatically add this as a `nativeBuildInputs`.

Reply:
Fixed